app/controllers/my_module_repositories_controller.rb

```diff
@@ -160,12 +160,19 @@ def consume_modal
   def update_consumption
     module_repository_row = @my_module.my_module_repository_rows.find_by(id: params[:module_row_id])
     module_repository_row.with_lock do
+      current_stock = module_repository_row.stock_consumption
       module_repository_row.assign_attributes(
         stock_consumption: params[:stock_consumption],
         last_modified_by: current_user,
         comment: params[:comment]
       )
       module_repository_row.save!
+
+      log_activity(current_stock,
+                   module_repository_row.stock_consumption,
```

Comment:
you can simply pass here module_repository_row object and than use it in log_activity method